### ISSUE TITLE
Plugin does not rotate back to first tab after going through the rest

### ISSUE THREAD

**Issue by mattie47** [KevinSheedy/chrome-tab-rotate#46]
Hi,

I've tried to use this plugin for a while in kiosk mode on an RPI3 and Chromium with a number of different tabs (about 10), however I've found that once it has rotated through all tabs one time, it does not go back and start again.

Any thoughts on why this may be?

**config:**
```
{
  "settingsReloadIntervalMinutes": 5,
  "fullscreen": true,
  "autoStart": true,
  "lazyLoadTabs": true,
  "websites": [
    {
      "url": "https://google.com",
      "duration": 360,
      "tabReloadIntervalSeconds": 30
    },
    {
      "url": "https://google.com",
      "duration": 360,
      "tabReloadIntervalSeconds": 30
    },
    {
      "url": "https://google.com",
      "duration": 360,
      "tabReloadIntervalSeconds": 30
    },
    {
      "url": "https://google.com",
      "duration": 360,
      "tabReloadIntervalSeconds": 30
    },
    {
      "url": "https://google.com",
      "duration": 360,
      "tabReloadIntervalSeconds": 30
    },
    {
      "url": "https://mixpanel.com",
      "duration": 360,
      "tabReloadIntervalSeconds": 30
    },
    {
      "url": "https://mixpanel.com/",
      "duration": 360,
      "tabReloadIntervalSeconds": 30
    },
    {
      "url": "https://mixpanel.com/",
      "duration": 360,
      "tabReloadIntervalSeconds": 30
    },
    {
      "url": "https://google.com",
      "duration": 360,
      "tabReloadIntervalSeconds": 30
	}
  ]
}
```

Thanks,

Matt

**Comment by dolik-rce** [comment]
Hi, I've hit the same issue on very similar setup (RaspberryPi 3B, Raspbian, Chromium) and very similar config (only diference is my didn't have `lazyLoadTabs` enabled).

Is there any information I can provide to help with debugging and fixing this issue?

**Comment by KevinSheedy** [comment]
If you could grab the browser console log, that would be very helpful.
* Right-click the TabRotate Icon
* Click **Manage Extensions**
* Inspect Views -> **Background Page**

**Comment by dolik-rce** [comment]
Hi Kevin, I'm sorry it took so long, but it just didn't want to crash while I watched it...

Full log is attached and also a screenshot of errors from extensions page. It seems that it locks after failed configuration request.

[tabrotate.log](https://github.com/KevinSheedy/chrome-tab-rotate/files/3895370/tabrotate.log)
![2019-11-27-070003_709x557_scrot](https://user-images.githubusercontent.com/365613/69698017-6f0e0880-10e4-11ea-9235-0c96fe09015f.png)

**Comment by KevinSheedy** [comment]
Thanks for uploading. Yes, it looks like the error handling is not robust enough and needs a code update.

**Comment by KevinSheedy** [comment]
Fixed in v0.4.1

**Comment by RichardWright** [comment]
@KevinSheedy How can this be used in normal chrome os in kiosk mode?
